django/template/base.py

```diff
@@ -150,13 +149,6 @@ def loader_name(self):
 
 class Template:
     def __init__(self, template_string, origin=None, name=None, engine=None):
-        try:
-            template_string = force_text(template_string)
-        except UnicodeDecodeError:
-            raise TemplateEncodingError(
```

Comment:
Did you intentionally keep the unused `TemplateEncodingError` for backwards compatibility? I think it could be removed.